src/rxn_network/reactions/basic.py

```diff
@@ -169,6 +189,27 @@ def normalized_repr(self):
         """
         return self.normalized_repr_and_factor()[0]
 
+    def get_vector(self, entry_indices, num_entries):
+        entry_indices = np.array(entry_indices)
+
+        vector = np.zeros(num_entries)
+        vector[entry_indices] = self._coefficients
+
+        return vector
```

Comment:
This shouldn't be here. It can be a helper, but it has no context with respect to a `Reaction` it matters, I'm assuming in the Network.

Reply:
Good catch. This is left over from before and I forgot to delete it.

Reply:
Fixed